modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -0,0 +1,64 @@
+---
+- job:
+    name: Deploy_App_Downstream
+    display-name: Deploy_App_Downstream
```

Comment:
v nit: any reason for this for this casing? Can we remove the underscores from display-name (I assume its for humans to read)?

Reply:
Seems we have a mishmash of casing styles used across our Jenkins jobs - might be clearer to stick to something more standard kebab or snake

Reply:
I was going for consistency with `Deploy_App` here. We seem to have two predominant styles (`_` or ` `). I actually think `_` is a better style, because it's the default and generally matches the name of the file (to look for) in puppet - freestyle less so.